src/core/src/op/constant.cpp

```diff
@@ -79,6 +79,14 @@ Constant::Constant(const std::shared_ptr<ngraph::runtime::Tensor>& tensor) {
         constructor_validate_and_infer_types();
         allocate_buffer(false);
         tensor->read(get_data_ptr_nc(), tensor->get_size_in_bytes());
```

Comment:
No, it should not because we have no other way to grab values from source tensor. So firstly, we need to read these objects to available buffer and re-initialize std::string objects with same values.

Reply:
in this case we iterate over array 2 times, but we can allocate and then copy from src array to dst one